src/timetravel/CallBreaker.sol

```diff
@@ -108,10 +104,9 @@ contract CallBreaker is CallBreakerStorage {
         uint256 amountA,
         address tokenB,
         uint256 amountB,
-        bytes calldata data
+        CallObject[] calldata calls
```

Comment:
I think we should keep this as bytes since it is called from a contract and not an EOA, and the called contract is a third party contract so they will always use the standard data types like `bytes`